cmd/load-runner/runner.go

```diff
@@ -16,7 +16,7 @@ import (
 	"rudder-load/internal/parser"
 )
 
-type helmClient interface {
+type infraClient interface {
```

Comment:
Since you're renaming this one I suggest renaming after its actions and instead of giving it a "thing" name. e.g. `releaseManager`. At least it would be a bit more specific. `infraClient` is so generic :D